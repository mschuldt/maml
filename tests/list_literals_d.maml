arduino = Arduino(desktop=True)

@block(once)
def test():
    x = [1,2,3,4]
    y = x
    while y:
        print(car(y))
        y = cdr(y)
    print(car(x))

    x = [test_print(11), test_print(33), test_print(33)]
    while x:
        print(car(x))
        x = cdr(x)
    die(1)

arduino.send(test)
