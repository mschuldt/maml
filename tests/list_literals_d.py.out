Initializing avm...
Ready.

1
2
3
4
1
-> 11
-> 33
-> 33
11
33
33
bye.
